concourse/scripts/bosh-cli.sh

```diff
@@ -10,12 +10,17 @@ export TARGET_CONCOURSE=deployer
 $("${SCRIPT_DIR}/environment.sh")
 "${SCRIPT_DIR}/fly_sync_and_login.sh"
 
-OUTPUT_FILE=$(mktemp -t bosh-cli.XXXXXX)
-trap 'rm -f "${OUTPUT_FILE}"' EXIT
+BUILD_NUMBER=$($FLY_CMD -t "${FLY_TARGET}" trigger-job -j create-cloudfoundry/bosh-cli | \
+	awk '/started create-cloudfoundry\/bosh-cli/ { print $3 }' | \
+	tr -d '#')
 
-$FLY_CMD -t "${FLY_TARGET}" trigger-job -j create-cloudfoundry/bosh-cli -w | tee "${OUTPUT_FILE}"
-
-BUILD_NUMBER=$(awk '/started create-cloudfoundry\/bosh-cli/ { print $3 }' "${OUTPUT_FILE}" | tr -d '#')
+while $FLY_CMD -t "${FLY_TARGET}" builds -j create-cloudfoundry/bosh-cli | awk '{print $3 $4}' | grep "${BUILD_NUMBER}pending" >/dev/null; do
+	echo "waiting for bosh-cli container to start..."
```

Comment:
It would help with debugging to include the the `BUILD_NUMBER` both here and below in the hijacking message.

This task failed the first time for me (I suspect due to office network issues), and debugging which build was involved was a little fiddly.

Reply:
🙌  done